core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -161,6 +161,7 @@ private[spark] object UIUtils extends Logging {
 
   def commonHeaderNodes: Seq[Node] = {
     <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
+    <link rel="shortcut icon" href="about:blank"/>
```

Comment:
The issues you're trying to address are generally harmless errors across the entire UI used by partial imports of 3rd party libraries, which is common in web dev. I'm still a fan of addressing them though.